components/grade-table.js

```diff
@@ -1,25 +1,53 @@
 class GradeTable {
-    constructor(tableElement){
+    constructor(tableElement,noGradesElement){
         this.tableElement = tableElement;
+        this.noGradesElement = noGradesElement;
     }
+
     updateGrades(grades){
-        console.log('grades: ',grades);
         var tbody = this.tableElement.querySelector('tbody');
         tbody.textContent = '';
+        this.renderGradeRow(grades,this.deleteGrade);
+        var p = document.querySelector('p');

```

Comment:
Rather than re-querying the whole `document`, why not use the reference you already saved in the `constructor`?